austation/code/modules/power/fuel_rods.dm

```diff
@@ -1,146 +1,291 @@
+
+/obj/item/twohanded/required/fuel_rod/material/telecrystal/Final_Depletion()
+	..()
+	fuel_power = 0.60 // thrice as powerful as plutonium, you'll want to get this one out quick!
+	name = "Exhausted Telecrystal Fuel Rod"
+	desc = "A highly energetic, disguised titanium sheathed rod containing a number of slots filled with greatly expanded telecrystals which can be removed by hand. It's extremely efficient as nuclear fuel, but will cause the reaction to get out of control if not properly utilised."
+	icon_state = "telecrystal_used"
+	AddComponent(/datum/component/radioactive, 3000, src)
+
+/obj/item/twohanded/required/fuel_rod/material/bluespace
+	name = "Bluespace Crystal Fuel Rod"
+	desc = "A heavy-duty low-grade fuel rod which fissiles much slower than standard counterparts. However, it's complex and sophisticated design allows insertion and utilisation of a certain material with special properties, namely bluespace crystals."
+	icon_state = "bluespace"
+	fuel_power = 0.05 // bluespace growth shouldn't be *that* interesting, isn't it?
+	rad_strength = 200
+	og_fuel_power = 0.05
+	max_initial_amount = 15
+	multiplier = 3
+	depletion_speed_modifier = 2
+	material_type = /obj/item/stack/sheet/bluespace_crystal
+	material_name = "bluespace crystals"
+	// bananium specific stuff
+	var/bananium_initial_amount = 0
+	var/bananium_slot = 2 // going double bananium won't hurt, I promise
+	var/bananium_grown_amount = 0
+
+/obj/item/twohanded/required/fuel_rod/material/bluespace/deplete(amount=0.035)
+	..()
+	if(bananium_initial_amount == bananium_slot && prob(10))
+		playsound(src, pick('sound/items/bikehorn.ogg', 'sound/misc/bikehorn_creepy.ogg'), 50) // HONK
+	fuel_power = max(og_fuel_power - (depletion - 100) / 500, 0)
+
+/obj/item/twohanded/required/fuel_rod/material/bluespace/Final_Depletion()
+	..()
+	name = "Fully Grown Bluespace Crystal Fuel Rod"
+	desc = "A heavy-duty low-grade fuel rod which fissiles much slower than standard counterparts. Its growth stage has finished, and its power now starts to wane. You also can harvest bluespace crystals which should've been multiplied by now."
+	if (bananium_initial_amount)
+		icon_state = "bluespace_bananium"
+	else
+		icon_state = "bluespace_used"
+	bananium_grown_amount = bananium_initial_amount * 2
+	rad_strength *= 5
+	og_fuel_power *= 2  
+	AddComponent(/datum/component/radioactive, rad_strength, src) // Rads only go up
+
+/obj/item/twohanded/required/fuel_rod/material/bluespace/proc/update_stats()
+	multiplier = 3 + bananium_initial_amount
+	og_fuel_power = 0.05 + initial_amount * 0.01 // 0.2 would be high-end cases
+	rad_strength = 200 + bananium_initial_amount * 100 + initial_amount * 10
+	if(!depleted_final)
+		fuel_power = og_fuel_power
+
+/obj/item/twohanded/required/fuel_rod/material/bluespace/expend()
+	..()
+	icon_state = "bluespace_harvested"
+	update_stats() // Return to baseline
+
+/obj/item/twohanded/required/fuel_rod/material/bluespace/attackby(obj/item/W, mob/user, params)
```

Comment:
ok so what I want you to do here is
completely remove the bluespace crystal replication and make this rod exclusively bananium. I want this for two main reasons;
a) it avoids this immense override to allow for two types of material
b) bluespace crystal replication is literally completely useless for the time investment required. bananium would be far more interesting and fun, and would make for a great clown gimmick.

Reply:
Ah yes, bananium